processing/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    // Enable latest features
+    "lib": ["ESNext", "DOM"],
+    "target": "ESNext",
+    "module": "ESNext",
+    "moduleDetection": "force",
+    "jsx": "react-jsx",
```

Comment:
```suggestion
```

Die Zeile kommt mir komisch vor. Vielleicht nochmal mit https://github.com/total-typescript/tsconfig/blob/main/tsc/no-dom/library.json abgleichen. "TotalTypeScript" ist mein Goto Place für TS Recommendations. Und wir sind hier ja eher in der "Library" als in der DOM Welt.